src/DGmethods/NumericalFluxes.jl

```diff
@@ -238,8 +236,33 @@ function numerical_flux_diffusive!(::CentralNumericalFluxDiffusive,
   fill!(F⁺, -zero(FT))
   flux_diffusive!(bl, Grad{S}(F⁺), state⁺, diff⁺, aux⁺, t)
 
-  Fᵀn .+= (F⁻ + F⁺)' * (n/2)
+  Fᵀn .+= (F⁻ + F⁺)' * (n⁻/2)
 end
 
+function numerical_boundary_flux_diffusive!(nf::CentralNumericalFluxDiffusive,
+    bl::BalanceLaw, fluxᵀn::Vars{S}, n⁻::SVector,
+    state⁻::Vars{S}, diff⁻::Vars{D}, aux⁻::Vars{A},
+    state⁺::Vars{S}, diff⁺::Vars{D}, aux⁺::Vars{A},
+    bctype, t,
+    state1⁻::Vars{S}, diff1⁻::Vars{D}, aux1⁻::Vars{A}) where {S,D,A}
+
+  FT = eltype(fluxᵀn)
+  nstate = num_state(bl,FT)
+  Fᵀn = parent(fluxᵀn)
+
+  F⁻ = similar(Fᵀn, Size(3, nstate))
+  fill!(F⁻, -zero(FT))
+  flux_diffusive!(bl, Grad{S}(F⁻), state⁻, diff⁻, aux⁻, t)
+
+  F⁺ = similar(Fᵀn, Size(3, nstate))
+  fill!(F⁺, -zero(FT))
+  boundary_flux_diffusive!(nf, bl,
```

Comment:
So if we want to prescribe a specific diffusive boundary flux, is this function we should extend?

Reply:
cc: @fxgiraldo

Reply:
That's my thought right now. Something like what I did in the [dycoms bc]

[dycoms bc]:  https://github.com/climate-machine/CLIMA/blob/26ce8f722f07cfaf8f97a4f4624ccf288a0bc0ca/src/Atmos/Model/boundaryconditions.jl#L126-L241

Reply:
This seems to conflict with the Numerics design docs:
<img width="614" alt="Screen Shot 2020-01-24 at 10 34 49 AM" src="https://user-images.githubusercontent.com/1692009/73094321-2f219280-3e95-11ea-8012-94e505cd2a82.png">

Reply:
I was just trying to preserve close to what we were currently doing, not to bring things in to conformity with Bassi and Rebay (which is our ultimate goal). So you are correct that we should directly overload the function.

Reply:
That's fine, I'm trying to figure out how things should be done.

Reply:
Do you want to BCs fixed before this is merged?

Reply:
I think if we merge this, then #630, I have a pretty good idea what we need to do for the BCs.

Reply:
We could also just merge the BCs update into #630 if you wanted (since that's the main motivation for #630)

Reply:
Lets get this in.

Reply:
Whatever you think is best: would be great to get them both wrapped up today.